wewatch-client/src/components/Lobby/Lobby.css

```diff
@@ -10,12 +14,24 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    background: #FFFFFF;
+    background: rgb(255, 255, 255,0.8);
```

Comment:
nitpick: space after the comma, ',0.8' -> ', 0.8' :)

Reply:
good catch, updated!